php/functions.php

```diff
@@ -47,30 +42,126 @@ function addParagraphs(array $paragraphs) :string {
  * @param $addSubmit string inserts the string in the text area to the database and pairing it with an `id` and a `deleted` number
  */
 function addParagraphToDb (PDO $db, string $addSubmit) : void{
-    $query = $db->prepare("INSERT INTO `about_me` (`paragraph`) VALUES (:newParagraph);");
-    $query->bindParam(':newParagraph', $addSubmit);
-    $query->execute();
+    if (empty(trim($addSubmit))){
+        header('Location: admin_page.php');
```

Comment:
Don't think you should have headers here in the functions redirecting you, particularly as there's nothing in the function name of docblock to suggest it redirects you

Reply:
removed this from function, added it into the admin_add in an if statement